### ISSUE TITLE
Gradle 8 Support

### ISSUE THREAD

**Issue by MarkRx** [gradle-plugins/toolbox#94]
Looks like Gradle 8.x isn't fully supported yet. It should get added.

`Unknown minimum Java version for Gradle '8.3.0', please open an issue on https://github.com/gradle-plugins/toolbox. Assuming value of the latest known version.`

See https://github.com/gradle-plugins/toolbox/blob/v1.6.9/subprojects/gradle-plugin-development/src/main/java/dev/gradleplugins/GradleRuntimeCompatibility.java

**Comment by MarkRx** [comment]
Looks like this is already in main. We just need a new release to pick it up.

**Comment by lacasseio** [comment]
You are right. I'm the changes are in but it wasn't released. I will try to get that release soon-ish.

**Comment by lacasseio** [comment]
I prepared a version with this change only. I expect to release it hopefully this week.

**Comment by lacasseio** [comment]
Support for all released versions and a fix to deprecation warning.

**Comment by lacasseio** [comment]
The message simply means that it assumes the Gradle dependencies are the same version from the latest "known" Gradle. They are generally compatible but they may differ. I will try to improve the wording of the message.

**Comment by ihostage** [comment]
Sorry, when I told about support 8.4, I mean to add info about 8.4 in `GradleRuntimeCompatibility`

**Comment by lacasseio** [comment]
I added the support and fixed deprecation warnings/failures when using the toolbox plugin on the latest Gradle. Everything is released as part of [v1.6.10](https://github.com/gradle-plugins/toolbox/releases/tag/v1.6.10).

**Comment by ihostage** [comment]
Thank a lot, Daniel! 👍
